ui/admin/tests/unit/controllers/scopes/scope/auth-methods/auth-method/accounts/account/index-test.js

```diff
@@ -1,5 +1,5 @@
 import { module, test } from 'qunit';
-import { setupTest } from 'admin/tests/helpers';
+import { setupTest } from 'ember-qunit';
```

Comment:
None blocking comment and for learninng purposes: I was not aware we need to perform this changes, should we consider a refactor or eliminate the helper we were using before? [link to code](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/tests/helpers/index.js#L41-L45).

I would like to learn a bit more about this change, do you have some context, perhaps link to documentation, on this change? Thank you!!

Reply:
ahh good catch! i didn't realize there was a helper there, when i auto-generated this test file it automatically imported from `admin/tests/helpers` and i got the message "Could not find a declaration file for module" so I changed it to what i saw another controller file was importing from. Now that you point this out I am noticing that tests pass either way, so I'll reverse this change.

Reply:
@lisbet-alvarez this is interesting, should we bring this topic to tech time and discuss the need for `admin/test/helpers` so far, what I understand of such helper is that is a wrapper but aside from wrapping we are doing nothing custom, so maybe we should rely on the default one until we have the need for customize?